src/pages/MyTimePage/MyTimePage.scss

```diff
@@ -49,3 +49,12 @@
   left: 4%;
   border-radius: 8px;
 }
+
+.scheduling-message {
```

Comment:
please use a custom class from short.scss for the font weight and font size prop